contents/javascript/Javascript-framework-Angular.md

```diff
@@ -193,19 +193,20 @@ In contrast, Angular provides data binding in both ways. That means even changes
 
 This [article](https://medium.com/@preethi.s/angular-custom-two-way-data-binding-3e618309d6c7) on Medium provides a good introduction to two-way data binding in Angular.
 
-### Disadvantages of Angular
+## Disadvantages of Angular
 
 Like any other framework/library, Angular has its share of disadvantages.
 
-1. **Steep Learning Curve** - Angular requires you to learn many concepts, such as _directives_, modules, components, services and many more. 
-The large number of new concepts can be confusing to newcomers. Additionally, Angular requires you to learn a new language, TypeScript.
+1. **Steep Learning Curve:** <br/>
+Angular requires you to learn many concepts, such as _directives_, modules, components, services and many more. The large number of new concepts can be confusing to newcomers. Additionally, Angular requires you to learn a new language, TypeScript.
 
-2. **Opinionated Framework** - Angular is opinionated about how you structure your code. This means that Angular provides you defaults for building applications. For instance, there are in-built services to perform data fetching, state management and much more. It is also opinionated about the development language as it requires you to use TypeScript.<br/>
+2. **Opinionated Framework:**<br/>
+Angular is opinionated about how you structure your code. This means that Angular provides you defaults for building applications. For instance, there are in-built services to perform data fetching, state management and much more. It is also opinionated about the development language as it requires you to use TypeScript.<br/>
```

Comment:
Both this and the preceding section mention that Angular favours the use of Typescript. Perhaps we can remove the duplication to keep things concise?

Reply:
Can do this in another PR.